specification/schemas/ResourceValue.json

```diff
@@ -0,0 +1,32 @@
+{
+  "type": "object",
+  "required": [
+    "value_type",
+    "value"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "value_type": "resource",
+    "value": {
+      "type": "object",
+      "required": [
+        "resource_type",
+        "attributes"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "resource_type": {
+          "type": "string",
+          "example": "regions"
+        },
+        "path": {
+          "type": "string",
+          "example": "relationships.parent.relationships.parent"
+        },
+        "attributes": {
+          "type": "object",
```

Comment:
The attributes in the proposal have dynamic properties. Now that I think about it, that may not be the best way as now you cannot define how these attributes should look, so this may need to be changed from the proposal.

Reply:
What did you have in mind? adding an oneOf and add all the resource types?